### PR TITLE
Return new property value after PUT.

### LINKED ISSUE #70: Web Thing REST API - Echo back value when setting property
An HTTP PUT request on a Property resource should echo back the value that was set in its response.

e.g.

```
PUT http://mythingserver.com/things/pi/properties/led
{
  "led": true
}
```

```
200 OK
{
  "led": true
}
```

### PR DESCRIPTION
**Pull request by mrstegeman**
Fixes #70

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -246,7 +246,11 @@ <h3><code>Property</code> resource</h3>
         <div class="example-title marker">
           Response
         </div>
-        <pre>200 OK</pre>
+        <pre>200 OK
+{
+  "led": true
+}
+        </pre>
```
Comment (hobinjk) [inline index.html]: This trailing space makes it look a bit off:
screen shot 2018-03-13 at 12 36 10 pm

Comment (hobinjk) [reply index.html]: (that last line is the "        ")

**Comment by benfrancis** [comment]
Fixes #70

**Comment by mrstegeman** [comment]
Space removed.

**Pull request merged**